server/api/user/user.model.js

```diff
@@ -18,9 +18,9 @@ var UserSchema = new Schema({
   hashedPassword: String,
   provider: String,
   salt: String,
-  tech: [String],
+  tech: [String], 
   projects: [{type : Schema.Types.ObjectId, ref: 'Project'}], // project id
-  bio:String,
+  bio: {type:String, default: "No Bio written yet. Please tell us some more about yourself"},
```

Comment:
@agundy should we do this client side or server side? My intuition says it might be better on the client, but this does solve the problem.